.circleci/config.yml

```diff
@@ -113,6 +130,7 @@ workflows:
           requires:
             - lint-scripts
             - lint-charts
+            - insights
```

Comment:
If insights fails, you want to not run end-to-end tests?  That seems like a lot of work for you! :-D

Reply:
also, to be certain, this won't disrupt other builds, right. Since you have the insights config in there, it will just pass if they aren't configured?

Reply:
Good point. Insights should always exit 0 (we'll have a separate status check in GH to pass/fail), but might as well not wait for it.